### PR TITLE
feature: what's new, diff between v1 and v2

### LINKED ISSUE #363: diff btw PE v1 and v2
what are the differences between Presentation Exchange v1 and PE v2?
Would it be possible to add a section with bulletpoints enumerating major differences?

### PR DESCRIPTION
**Pull request by nklomp**
This PR lists the changes between v1 and v2 of the spec
closes #363

### CODE REVIEW

**Comment by dtmcg** [comment]
@nklomp is this ready for review ?

**Comment by nklomp** [comment]
Yes. I was under the impression I marked it as ready

**Review by brentzundel — DISMISSED** [review]

**Comment by andorsk** [comment]
To update from: https://github.com/decentralized-identity/presentation-exchange/pull/360

File: spec/spec.md [hunk 1]
```diff
@@ -1553,6 +1575,54 @@ JSONPath                      | Description
...
+Also the [[ref:Input Descriptor Object]] now may contain a `format` property, next to the toplevel `format` property, to constrain a submission of a single object to a subset of formats.
+
+#### The Input Descriptor constraint property has been made mandatory
+The [[ref:Input Descriptor Object]] now must contain a `constraints` property, where it was optional before. Given other optional fields, omission of the `constraints` property could potentially lead to an [[ref:Input Descriptor Object]] that didn't express anything actionable anymore. 
+
+#### More concise language for consumers and producers
+Language around consumers and producers of [[def:Presentation Definition]] and [[def:Presentation Submission]] objects has been made more concise to prevent confusion. 
+
+
```
Comment (brentzundel) [inline spec/spec.md:1625]:
```suggestion
#### `name` and `description` properties added to the spec.
```

**Review by csuwildcat — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Pull request merged**